Conceptpower+Spring/src/main/java/edu/asu/conceptpower/core/impl/DeleteConceptManager.java

```diff
@@ -0,0 +1,16 @@
+package edu.asu.conceptpower.core.impl;
+
+import org.springframework.stereotype.Service;
+
+import edu.asu.conceptpower.core.ConceptEntry;
+import edu.asu.conceptpower.core.IDeleteConceptManager;
+
+@Service
+public class DeleteConceptManager  implements IDeleteConceptManager{
+
+    @Override
+    public void setDelete(ConceptEntry entry) {
```

Comment:
this method should be part of the conceptmanger. dont' create a sepearte class, just add it to the exsiting one.